src/dodal/devices/shutter.py

```diff
@@ -1,21 +1,41 @@
+import asyncio
 from enum import Enum
 
-from ophyd_async.core import StandardReadable
-from ophyd_async.epics.signal import epics_signal_rw
+from bluesky.protocols import Movable
+from ophyd_async.core import AsyncStatus, StandardReadable, wait_for_value
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_w
 
 
-class OpenState(str, Enum):
+class OpenStateSet(str, Enum):
+    CLOSED = "close"
+    OPEN = "open"
```

Comment:
This needs reset

Reply:
what would be the logic for this then?

Reply:
added, with an early return logic for 'set' comparison for readback